src/pages/index.md

```diff
@@ -9,7 +9,7 @@ hero:
   bg_video: images/home/keep-castle-clouds.mp4
   bg_image: /images/hero-bg.jpg
   cta_buttons:
-    - label: Mint tBTC
+    - label: 'Mint <span style="text-transform: lowercase">t</span>BTC'
       url: https://threshold.network/earn/btc
```

Comment:
Shouldn't this button navigate to `https://dashboard.threshold.network/tBTC` instead of `https://threshold.network/earn/btc/`? We have three `Mint tBTC` buttons in the website and only this one navigates to the `threshold.network` instead of the dApp. What do you think?

Reply:
🤷 We can always update via cms.